src/jgikbase/test/idmapping/service/mapper_service_test.py

```diff
@@ -239,8 +239,8 @@ def test_root_and_logging():
     assert_ms_epoch_close_to_now(time_)
     assert resp.status_code == 200
 
-    assert len(logstream.write.call_args_list) == 2
-    assert logstream.write.call_args_list[1][0][0] == '\n'
+    assert len(logstream.write.call_args_list) == 1
+    assert logstream.write.call_args_list[0][0][0][-1] == '\n'
```

Comment:
Why do the logs change in these tests?

Reply:
python 3.6 -> 3.9.19

Reply:
What is the difference in the logs?

Reply:
Original logstream.write.call_args_list: 
```
[call(‘{“service”: “IDMappingService”, “level”: “INFO”, “time”: 1719870386183, “source”: “jgikbase.idmapping.service.m...“, “ip”: “127.0.0.1", “method”: “GET”, “callid”: “8909603642060648", “msg”: “GET / 200 werkzeug/2.0.3"}‘),\n call(‘\n’)]
```

Current logstream.write.call_args_list:
```
[call('{"service": "IDMappingService", "level": "INFO", "time": 1718735741424, "source": "jgikbase.idmapping.service.mapper_service", "ip": "127.0.0.1", "method": "GET", "callid": "2489321465379915", "msg": "GET / 200 Werkzeug/3.0.3"}\n')]
```

Reply:
Based on above diff in logstream.write.call_args_list, need to change len and index accordingly

Reply:
I see, so it's omitting the blank line

Reply:
Ok, in that case I think you can just get rid of the lines checking that the last character of the line is a newline, that's not a valuable tests AFAICT

Reply:
Why is werkzeug listed as 3.0.3?

Reply:
That was log from an early commit. I can display the most recent one where werkzeug is 2.0.3.

Reply:
:+1: